src/components/Map/Map.tsx

```diff
@@ -0,0 +1,109 @@
+import type Locale from "@/locales";
+import Button from "./Button";
+import { SvgLoader, SvgProxy } from "react-svgmt";
+import { Dispatch, useEffect, useState } from "react";
+
+function range(start: number, end: number) {
+  return [...Array(1 + end - start).keys()].map((v) => start + v);
+}
+
+function circleSvgProxy(position: number, selectedExhibitor: number) {
+  return (
+    <SvgProxy
+      key={position}
+      selector={`#p${position}`}
+      class={
+        selectedExhibitor === position
+          ? "selectedStyle"
```

Comment:
```suggestion
          ? "cursor-pointer z-50 animate-[blinkingText_2s_infinite] drop-shadow-[0_0_7px_#ee2a7b]"
```

Reply:
I think we should either implement this in tailwind as my suggestion above or give the class a much more descriptive name.